python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -95,9 +97,20 @@ class DiracBase(IBackend):
     _hidden = True
 
     def _setup_subjob_dataset(self, dataset):
+        """
+        Unknown what this ethod does other than return None:
```

Comment:
This is not a helpful docstring. Looking at its overload in `Dirac` would tell you what it does.